nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/csv/TestCSVHeaderSchemaStrategy.java

```diff
@@ -66,9 +66,37 @@ public void testSimple() throws SchemaNotFoundException, IOException {
             .allMatch(field -> field.getDataType().equals(RecordFieldType.STRING.getDataType())));
     }
 
```

Comment:
which is this testing, Jackson or Apache?

Reply:
The "Use String Fields From Header" Schema Access Strategy is parser-agnostic (as far as I can tell) in the CSVReader service, however the implementation is hard-coded to use the Apache Commons CSV Parser (I've not changed that).

That parser sets up its own CSV Format using the same (updated) CSVUtils methods, so it should take into account the settings changes.

Reply:
Is there someplace where are are parser specific tests?  It seems we would want  to explicitly test each